features/badging.yml.dist

```diff
@@ -0,0 +1,22 @@
+# Generated from: badging.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: false
+  support:
+    safari: "17"
+    safari_ios: "16.4"
```

Comment:
This is unfortunate and due to the fact that BCD shows Chromium as partially implemented only.
Badging does work on Chrome and Edge, for installed apps on Windows and macOS, and is documented in multiple places. So, having web-features saying otherwise is a bit of a shame.

@foolip do you have any suggestions? How have we delt with this in the past? Should we allow some wiggle room for editorial overrides for this type of things where, yes, it's not all green, but making it red gives the wrong idea.

Reply:
So the problem is here in BCD:

https://github.com/mdn/browser-compat-data/blob/3b8d8ea67b2a4bd03e8b1aaac80cb54475e221e8/api/Navigator.json#L4229-L4233

And using `partial_implementation` there is in line with BCD guidelines:
https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/index.md#operating-system-limitations-imply-partial_implementation

This is very similar to Web Bluetooth, which is also not supported on Linux. I used a status override for that in web-features:

https://github.com/web-platform-dx/web-features/blob/9189c4c05dbccf2b930cc23cd63c05198c033064/features/web-bluetooth.yml#L19-L21

I think we can do that here too.

Unfortunately, the dist files still have the data derived from BCD, blessed as correct:

https://github.com/web-platform-dx/web-features/blob/9189c4c05dbccf2b930cc23cd63c05198c033064/features/web-bluetooth.yml.dist#L5-L7

@ddbeck any ideas for how we can solve this last problem?

Reply:
OK, so there's some distinctions to make here:

- The comments don't follow the override. Would it be helpful if they _did_? Or at least made it clear that the overall status wasn't derived from what's shown in the comments?

- Do we want to override the per-key statuses, to the value of the parent feature override? Or do we want to override the BCD key statuses, then use that to calculate the full feature status?

- If we do want to be able to override per-key statuses, do want want _that_ to happen on a per-feature basis (e.g., it's part of the feature authoring and the scope is limited to that feature)? Or on a per-key basis (e.g., we override BCD once and for all, even once we solve the keys in multiple features problem)?

Reply:
I filed https://github.com/web-platform-dx/web-features/issues/1690.